src/analyze/analyze-security.c

```diff
@@ -176,7 +178,8 @@ static int assess_user(
         assert(ret_description);
 
         if (streq_ptr(info->user, NOBODY_USER_NAME)) {
-                d = strdup("Service runs under as '" NOBODY_USER_NAME "' user, which should not be used for services");
+                d = strdup("Service runs under as '" NOBODY_USER_NAME
+                           "' user, which should not be used for services");
```

Comment:
It's preferable to keep such strings on one line to make grepping easier. Please exclude.

Reply:
line wrapping forces this to the new line. Excludable by clang-format off/on